### PR TITLE
fix(Loader): remove inline css and refactor component

### PR DESCRIPTION
**Pull request by MartinWeb**
### Reference to the issue

https://github.com/AxaGuilDEv/react-toolkit/issues/962

### Description of the issue

`When using Loader component, we can add a className to change the style of the component (height, background, ...), but as the style is inline in the loader component we are forced to put !important on our style to override the one of the toolkit. `

Little point of attention : 
When applying a className we loose style that is defined on the component. It's a problem that we can have on other components of the toolkit, so i don't see it as an issue, but i am open to review on that.

### Person(s) for reviewing proposed changes

@arnaudforaison @samuel-gomez

### CODE REVIEW

File: packages/loader/src/spinner.scss [hunk 1]
```diff
@@ -1,5 +1,19 @@
 @import '@axa-fr/react-toolkit-core/src/common/scss/core.scss';
 
+.af-loader {
+  min-height: 94px;
+  position: relative;
+
+  .af-loader-on {
```
Comment (samuel-gomez) [inline packages/loader/src/spinner.scss:7]: tu peux utiliser le '&' 
```suggestion
  &-on {
```

File: packages/loader/src/Loader.tsx [hunk 2]
```diff
@@ -16,44 +15,29 @@ const texts: { [index: string]: string } = {
...
-            left: 0,
-            width: '100%',
-            height: '100%',
-          }}>
-          <div
-            className={`af-spinner ${
-              isLoaderVisible ? 'af-spinner--active' : ''
-            }`}>
+        <div className={`${className} af-loader-on`}>
+          <div className="af-spinner af-spinner--active">
```
Comment (samuel-gomez) [inline packages/loader/src/Loader.tsx:40]: je pense qu'avoir un modifier ici, n'est plus utile puis que tu as remonté la condition plus haut

Comment (MartinWeb) [reply packages/loader/src/Loader.tsx:40]: @samuel-gomez tu veux dire de supprimer la classe af-spinner--active et rapatrier son style directement dans af-spinner ?

File: packages/loader/src/__tests__/Loader.spec.tsx [hunk 3]
```diff
@@ -5,14 +5,52 @@ import LoaderModes from '../LoaderModes';
...
     );
-    expect(asFragment()).toMatchSnapshot();
+    expect(queryByText('Chargement en cours')).not.toBeNull();
+    expect(queryByText('Title Child')).not.toBeNull();
+
+    const animation = document.getElementsByClassName('af-spinner__animation');
+    expect(animation.length).toBe(1);
+  });
+
+  it('renders Loader with none mode', () => {
```
[the remark this answers is not in the document]
Comment (MartinWeb) [reply packages/loader/src/__tests__/Loader.spec.tsx:23]: Tout à fait d'accord, je fais ça.

File: packages/loader/src/Loader.tsx (hunk 2 above)
Comment (MartinWeb) [reply packages/loader/src/Loader.tsx]: Pris en compte.

**Review by samuel-gomez-axa — APPROVED** [review]

**Review by arnaudforaison — APPROVED** [review]
ATTENTION, ce changement de classe CSS est un breaking change important pour tous ceux

**Pull request merged**